{"uuid": "669ef640-1e05-4b74-87eb-3b66db1f9608", "children": ["f7823a9b-3b4a-41f0-b97a-e5bd8456edbc"], "befores": [{"name": "setup", "status": "passed", "start": 1745418740240, "stop": 1745418745870}], "afters": [{"name": "setup::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 231, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\tests\\conftest.py\", line 149, in setup\n    driver.quit()\n    ~~~~~~~~~~~^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 193, in quit\n    self.service.stop()\n    ~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 157, in stop\n    self.send_remote_shutdown_command()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 142, in send_remote_shutdown_command\n    if not self.is_connectable():\n           ~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 131, in is_connectable\n    return utils.is_connectable(self.port)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Taurus13\\PycharmProjects\\PythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 101, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n  File \"C:\\Python\\Lib\\socket.py\", line 856, in create_connection\n    exceptions.clear()  # raise only the last error\n    ~~~~~~~~~~~~~~~~^^\n"}, "start": 1745418755876, "stop": 1745418757998}], "start": 1745418740240, "stop": 1745418757998}